scripts/format_notebooks.py

```diff
@@ -170,12 +174,23 @@ def preprocess_cell(self, cell, resources, cell_index):
         action="store_true",
         help="Perform default formatting, equivalent to -wcnk",
     )
-    parser.add_argument(
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument(
         "-o",
         "--overwrite",
         action="store_true",
         help="Overwrite original notebooks, otherwise a copy will be made with a .mod suffix",
     )
+    group.add_argument(
+        "--check",
```

Comment:
I'm not so sure; I wouldn't expect us to be running `--check` that often, since it seems more likely that we'd be running with `--overwrite` locally to do the formatting; it's pretty safe and idempotent, so I don't think we need to encourage "check -> format if failed" vs. only "format", since the latter is 2&times; faster.